src/evo/dmnstate.h

```diff
@@ -52,16 +99,42 @@ class CDeterministicMNState
     CScript scriptPayout;
     CScript scriptOperatorPayout;
 
+    uint160 platformNodeID{};
+    uint16_t platformP2PPort{0};
+    uint16_t platformHTTPPort{0};
+
 public:
     CDeterministicMNState() = default;
     explicit CDeterministicMNState(const CProRegTx& proTx) :
             keyIDOwner(proTx.keyIDOwner),
             keyIDVoting(proTx.keyIDVoting),
             addr(proTx.addr),
-            scriptPayout(proTx.scriptPayout)
+            scriptPayout(proTx.scriptPayout),
+            platformNodeID(proTx.platformNodeID),
+            platformP2PPort(proTx.platformP2PPort),
+            platformHTTPPort(proTx.platformHTTPPort)
     {
         pubKeyOperator.Set(proTx.pubKeyOperator);
     }
+    explicit CDeterministicMNState(const CDeterministicMNState_Oldformat& s) :
+            nPoSeBanHeight(s.nPoSeBanHeight),
+            nRegisteredHeight(s.nRegisteredHeight),
+            nLastPaidHeight(s.nLastPaidHeight),
+            nConsecutivePayments(0),
```

Comment:
~not needed, ~ ~~well, why not copy this over?~~ oh, because it's old format. In that case, not needed

Reply:
@PastaPastaPasta Of course it it is needed.
When migrating evoDB, first we read an `CDeterministicMNState_Oldformat` and then we construct `CDeterministicMNState` from `CDeterministicMNState_Oldformat`. So we need to copy all fields that are in common and set to default value the new fields.

Reply:
I specifically mean setting it to 0 isn't needed. B/c the default constructor / default value for the variable does it

Reply:
Gotcha